tools/gbench/report.py

```diff
@@ -437,7 +632,7 @@ def test_utest(self):
         ]
         json1, json2 = self.load_results()
         output_lines_with_header = generate_difference_report(
-            json1, json2, utest=True, utest_alpha=0.05, use_color=False)
+            json1, json2, utest=True, utest_alpha=UTEST_DEFAULT_ALPHA, use_color=False)
```

Comment:
Ah wait, I'm not sure it is a good idea to change this.
This is a test, and it is very specifically testing what happens with `utest_alpha=0.05`.
If you pass `UTEST_DEFAULT_ALPHA`, and then change it, the test will need updating.

Reply:
Alright, it makes sense. Then I think reverting the whole change makes most sense because most occurrences of it are in unit tests. Do you agree?

Reply:
I have re-read the diff, and i'm not sure what is cleaner.
Out of all the places where `0.05` is used, only one can be replaced with `UTEST_DEFAULT_ALPHA`.
So yes, the situation will not be great either way. I'd undo these `UTEST_DEFAULT_ALPHA` changes. :/

Reply:
Agreed. I will revert the change.